### ISSUE TITLE
using {ProjectDir} under Docker block in launchSettings.json does not work

### ISSUE THREAD

**Issue by Erezinho** [MicrosoftDocs/visualstudio-docs#8177]
I was trying to use $(pwd) as suggested before and now with {ProjectDir} but it still does not work unfortunately.
(I've upgraded to 17.3.0 Preview 2.0 - https://docs.microsoft.com/en-us/visualstudio/containers/container-launch-settings?view=vs-2022) 

The token is not replaced and docker daemon yields an error 

launchSettings.json:
```
...
"DockerfileRunArguments": "-v {ProjectDir}/subFolder/appsettings.json:/app/appsettings.json "
...
```

_docker: Error response from daemon: create {ProjectDir}/subFolder/appsettings.json: "{ProjectDir}/subFolder/appsettings.json" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intended to pass a host directory, use absolute path._

Tried also with $(ProjectDir)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b0eb0463-50c5-9697-2eaf-a5f5fcb01513
* Version Independent ID: bf27ddf9-e6a3-0717-1d3c-620c054f0d5c
* Content: [Visual Studio Container Tools launch settings - Visual Studio (Windows)](https://docs.microsoft.com/en-us/visualstudio/containers/container-launch-settings?view=vs-2022)
* Content Source: [docs/containers/container-launch-settings.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/main/docs/containers/container-launch-settings.md)
* Product: **visual-studio-windows**
* Technology: **vs-container-tools**
* GitHub Login: @ghogen
* Microsoft Alias: **ghogen**

**Comment by WilliamAntonRohm** [comment]
@Erezinho -- Erez, thank you for your feedback.  Please consider these resources and let me know if they help solve this issue:
- [Visual Studio on Microsoft Q&A](https://docs.microsoft.com/answers/products/vs) 
- [Visual Studio MSDN Forum](https://social.msdn.microsoft.com/Forums/vstudio) 
- [Stack Overflow](https://stackoverflow.com/search?q=%22visual+studio%22)  
- [Reddit](https://www.reddit.com/r/VisualStudio)

**Comment by Erezinho** [comment]
Hi William, 
I went over many posts - unfortunately nothing helps

**Comment by WilliamAntonRohm** [comment]
@Erezinho -- Erez, thank you for your follow-up.

@ghogen -- Gordon, please look into this potential documentation issue.

**Comment by ghogen** [comment]
@Erezinho 
This works for me on the 17.3 Preview 3 which should be released soon. It might have been an error to say it was released in 17.3 Preview 2.

**Comment by Erezinho** [comment]
Thank you @ghogen 
I'll even wait for official release as other team members need this (but I'm the only one with preview version)
